PlaceLandmarkGrid/PlaceLandmarkGrid.py

```diff
@@ -292,10 +314,12 @@ def onSampleRateChanged(self):
 
     def onSelect(self):
         self.createGridButton.enabled = bool(self.modelSelector.currentNode())
-        self.createGridFromPointsButton.enabled = bool(self.modelSelector.currentNode())
+        self.createGridFromPointsButton.enabled = bool(self.modelSelector.currentNode() and self.fiducialSelector.currentNode())
```

Comment:
`self.modelSelector.currentNode() is not None and self.fiducialSelector.currentNode() is not None`. Change it accordingly just below and above as well please.
(Btw what you did here in this line is wrong. Probably you took the above line as example. But following that logic, it would be `bool(self.modelSelector.currentNode()) and bool(self.fiducialSelector.currentNode())`. It would be ugly but correct. What you did is ugly and incorrect)